tests/integration/src/test/resources/python/app_mock.py

```diff
@@ -90,6 +92,34 @@ def check_health(self):
             logging.debug("Done processing health request.")
             return
 
+        def handle_ipc_shm_info(self):
```

Comment:
Could you document this method. It does not seem trivial to me.

Reply:
I've added some documentation, including an example output of the command, so it's easier to understand the regex